playground/backend/internal/preparators/python_preparators.go

```diff
@@ -15,7 +15,82 @@
 
 package preparators
 
+import (
+	"beam.apache.org/playground/backend/internal/logger"
+	"bufio"
+	"io"
+	"os"
+)
+
+const (
+	addLogHandlerCode = "import logging\nlogging.basicConfig(\n    level=logging.DEBUG,\n    format=\"%(asctime)s [%(levelname)s] %(message)s\",\n    handlers=[\n        logging.FileHandler(\"logs.log\"),\n    ]\n)\n"
+)
+
 // GetPythonPreparators returns preparation methods that should be applied to Python code
 func GetPythonPreparators(filePath string) *[]Preparator {
-	return &[]Preparator{}
+	addLogHandler := Preparator{
+		Prepare: addToCode,
+		Args:    []interface{}{filePath, addLogHandlerCode},
+	}
+	return &[]Preparator{addLogHandler}
+}
+
+// addToCode processes file by filePath and adds additional code
```

Comment:
not very clear from the description - what is the additional code? In what cases do we need extra code?

Reply:
addCodeToFile might be clearer? :)

Reply:
seems like we can union this function with writeCodeToFile

Reply:
and with writeToFile :)

Reply:
@ilya-kozyrev probably, we can join `writeCodeToFile` with `writeToFile`, but I don't think that this is a good idea to join `addToCode` and `writeCodeToFile`. `writeCodeToFile` contains a nicely extracted dedicated logic and greatly increases readability of `addToCode` method.

Reply:
`writeToFile(to *os.File, str string)` is a method that writes some `str` to file `to`. I need to use this function 2 times:
- write additional code that will configure the python logger.
- write all code from the original file to the temporary file.

So that's why I didn't join methods `writeToFile()` with `writeCodeToFile` (just avoid duplication).

Reply:
But I agree with renaming `addToCode`-> `addCodeToFile`.